### PR TITLE
Fixes #9502: Drop old protocol and acl for agents

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9502

### CODE REVIEW

**Comment by amousset** [comment]
Do not merge for now, I still need to test the real effects of maproot.

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/cf-served.st [hunk 1]
```diff
@@ -159,93 +140,41 @@ bundle common def
 
     policy_server::
       "acl" slist => {
-      "127.0.0.0/8" , "::1",
+        "127.0.0.0/8" , "::1",
 &if(AUTHORIZED_NETWORKS)&
-      host2ip("${def.policy_server}"), # the policy server can connect to a relay
-      &AUTHORIZED_NETWORKS:{net|"&net&",}&
+        host2ip("${def.policy_server}"), # the policy server can connect to a relay
+        &AUTHORIZED_NETWORKS:{net|"&net&",}&
```
Comment (peckpeck) [inline techniques/system/common/1.0/cf-served.st]: this block is not properly indented

**Comment by amousset** [comment]
Commit modified

**Pull request merged**